lua/autorun/server/sv_cfc_contact_forms.lua

```diff
@@ -170,6 +172,15 @@ local function submitFormForPlayer( data, endpoint, ply )
     )
 
     recordPlayerSubmission( ply )
+
+    for _,p in ipairs( player.GetHumans() ) do
+        if p:IsAdmin() then
+            net.Start( "Admin_alert" )
+            net.WriteString( plyName ) -- Writes the reporter's name
```

Comment:
Instead of writing the color and name of the player you could simply write the player as an entity, so net.WriteEntity( ply ) and then get the color and name on clientside.

Reply:
Changed to only use `net.WriteEntity( ply )` .